x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/ml/text_expansion_search_sparse_vector.yml

```diff
@@ -0,0 +1,109 @@
+# This test uses the simple model defined in
+# TextExpansionQueryIT.java to create the token weights.
+setup:
+  - skip:
+      features: headers
+  - do:
```

Comment:
This is only available since 8.11, I think you need to skip older versions.

Reply:
Thanks for the catch! 👍